src/types/actions/index.ts

```diff
@@ -34,6 +36,8 @@ export type SlackAction = BlockAction | InteractiveMessage | DialogSubmitAction
  * this case `ElementAction` must extend `BasicElementAction`.
  */
 export interface SlackActionMiddlewareArgs<Action extends SlackAction = SlackAction> {
+  logger: Logger;
```

Comment:
instead of adding `logger` and `client` properties to this all the other `AnyMiddlewareArgs` union members, i suggest that we add them to this line: https://github.com/slackapi/bolt/blob/3d59cf78999934dd81d8d18f4e514e24d001279c/src/types/middleware.ts#L24

this is where `context` and `next` are added to arguments, and allows us not to repeat these properties which have no dependence on the type of event that's being triggered.

Reply:
Thank you very much 🙇I'll update the changes with your suggestion!

Reply:
Ah I should have done this. Will do today.

Reply:
Done!